docs/06-contributors/99-maintainers/050-operations.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Operations Overview
+id: operations
+keywords: [operations]
+---
+
+## Coverage
+
+"Wing" encompasses several production assets; Some are directly user-facing, while others only affect contributors to Wing's ecosystem.
+Both are important to the health of the Wing however users should be prioritized over contributors when everything else is equal.
+
+User-facing projects and their supporting contributor pieces:
+
+- [Homepage](https://winglang.io)
+- [Documentation](https://docs.winglang.io)
+  - https://github.com/winglang/docsite
+  - https://github.com/winglang/wing/tree/main/docs
+- Wing CLI
+  - https://github.com/winglang/wing
+- Wing Console 
+  - https://github.com/winglang/console
+- [Playground](https://play.winglang.io)
+  - https://github.com/winglang/playground
+- [NPM Packages](https://www.npmjs.com/~monabot) (https://www.npmjs.com/package/winglang and https://www.npmjs.com/package/@winglang/sdk)
+- [VSCode Extension](https://marketplace.visualstudio.com/items?itemName=Monada.vscode-wing) 
+- [Wing Examples](https://github.com/winglang/examples)
+
+## Monitoring
+
+Monitoring may take one of two forms:
+
+- Polling monitors via BetterUptime check the status of GitHub actions and hosted sites to make sure they are currently reporting successfully
+  - "GitHub Action" checks includes builds, release, and other scheduled jobs that provide additional health verification
+- Manual reports from users via Slack or Github Issues
+
+Manual reports also must be manually escalated to become actual incidents by any available maintainers.
+Ideally manual reports should be minimized, and we should constantly look for ways to automate the detection of issues.
```

Comment:
Do we currently have automatic monitoring for all the assets?

If not, let's list the gaps and create an issue/s to cover before beta.